common-util/functions/requests.jsx

```diff
@@ -10,10 +10,10 @@ export const getEstimatedGasLimit = async (fn, account) => {
 
   try {
     const estimatedGas = await fn.estimateGas({ from: account });
-    return Math.floor(estimatedGas);
+    return Math.floor(estimatedGas * 1.1);
```

Comment:
Is there a magic number of 1.1? Is it sufficient for the overlap of rounding?

Reply:
Good point.. 
Perhaps we should use `Math.ceil` instead and be more generous with the multiplier?
@kupermind

Reply:
Ceil with the factor of 2 maybe?

Reply:
when confirming a tx, in metamask the provided gas limit is displayed, if use `Math.ceil(estimatedGas * 2)` - this will display gas estimation increased twice, probably making users think "why so hight?"

is my understanding correct that this is what we want? 
<img width="318" alt="image" src="https://github.com/valory-xyz/autonolas-contribution-service-frontend/assets/22725775/51f6aeae-9540-43f4-a3c9-b28343bb671d">

Reply:
Right, this is correct. I confused ceil with floor ). Anyway, we can go with the bridging approach where the sensible addition of the gasLimit is 20%, or a factor of 1.2.

Reply:
updated, hope that's what's expected 🙏